test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -24,22 +25,22 @@ public class HubConnectionTests
         {
             var connection = new Mock<IConnection>();
             connection.SetupGet(p => p.Features).Returns(new FeatureCollection());
-            connection.Setup(m => m.StartAsync()).Returns(Task.CompletedTask).Verifiable();
+            connection.Setup(m => m.StartAsync(new CancellationToken())).Returns(Task.CompletedTask).Verifiable();
             var hubConnection = new HubConnection(connection.Object, Mock.Of<IHubProtocol>(), null);
             await hubConnection.StartAsync();
 
-            connection.Verify(c => c.StartAsync(), Times.Once());
+            connection.Verify(c => c.StartAsync(new CancellationToken()), Times.Once());
```

Comment:
I think it passes by accident (if it does) - you need yo use `It.IsAny<CancellationToken>()` here as well